test/test_integration.py

```diff
@@ -48,20 +78,49 @@ def get_script_module(script_name):
     return imp.load_module(script_name, file, pathname, desc)
 
 mysql_engine, postgres_engine, sqlite_engine, msaccess_engine, csv_engine, download_engine, json_engine, xml_engine = ENGINE_LIST()
-csv_engine.opts = {'engine': 'csv', 'table_name': './{db}_{table}.txt'}
 
 
-def test_csv_from_csv():
-    simple_csv_module = get_script_module('simple_csv')
-    simple_csv_module.SCRIPT.download(csv_engine)
-    simple_csv_module.SCRIPT.engine.disconnect()
-    obs_out = file_2string("simple_csv_simple_csv.txt")
-    assert obs_out == simple_csv['expect_out']
+@pytest.mark.parametrize("dataset, expected",test_parameters)
+def test_csv_integration(dataset, expected, tmpdir):
+    csv_engine.opts = {'engine': 'csv', 'table_name': '{db}_{table}'}
+    assert get_csv_string(dataset, csv_engine, tmpdir, db=dataset["name"]) == expected
+
+
+@pytest.mark.parametrize("dataset, expected",test_parameters)
+def test_sqlite_integration(dataset, expected, tmpdir):
+    dbfile = os.path.normpath(os.path.join(os.getcwd(), 'testdb.sqlite'))
+    sqlite_engine.opts = {'engine': 'sqlite', 'file': dbfile, 'table_name': '{db}_{table}'}
+    assert get_csv_string(dataset, sqlite_engine, tmpdir, dataset["name"]) == expected
+    os.system("rm testdb.sqlite")
```

Comment:
We may want to to modify the order here a little bit to make sure that `testdb.sqlite`  gets removed even if the test fails. E.g.,

```
observed = get_csv_string(dataset, sqlite_engine, tmpdir, dataset["name"])
os.system("rm testdb.sqlite")
assert observed == expected
```

It does make this test look a little different than the others, but since we know that not deleting the SQLite db can cause issues with the tests it might be worth it.